File: app/src/main/java/com/test/xyz/demo/domain/model/weather/Astronomy.java

package com.test.xyz.demo.domain.model.weather;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Astronomy {

    @SerializedName("sunrise")
    @Expose
    public String sunrise;
    @SerializedName("sunset")
    @Expose
    public String sunset;

}

File: app/src/main/kotlin/com/test/xyz/demo/domain/model/weather/Astronomy.kt
package com.test.xyz.demo.domain.model.weather

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Astronomy {
    @SerializedName("sunrise")

    @Expose
    var sunrise: String? = null

    @SerializedName("sunset")
    @Expose
    var sunset: String? = null
}
